docs/advanced/api/authentication.rst

```diff
@@ -0,0 +1,63 @@
+==============
+Authentication
+==============
+
+The Flocker REST API uses TLS certification to secure and authenticate requests.
+This ensures an API request is both encrypted and verified to have come from an authorised user, while the corresponding response is verified to have come from the genuine cluster control service.
+
+Unlike regular HTTPS, Flocker uses TLS not just for establishing a secure connection between an API user and the cluster's control service, but also for identity verification; that is, client certification is used in place of a username and password to identify an API user to the cluster.
+
```

Comment:
Combine the above two sentences into one.  Something brief like:

"Certificates are used for both client and server authentication, entirely replacing the use of usernames and passwords commonly used in HTTPS."